client/index.js

```diff
@@ -2,12 +2,16 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { renderRoutes } from 'react-router-config';
 import { BrowserRouter } from 'react-router-dom';
-import routes from './routes';
+import { Provider } from 'react-redux';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import routes from './routes';
+import store from './store';
 
 ReactDOM.render(
-    <BrowserRouter>
-        <div>{renderRoutes(routes)}</div>
-    </BrowserRouter>,
+    <Provider store={store}>
+        <BrowserRouter>
+            <div>{renderRoutes(routes)}</div>
```

Comment:
According to the readme examples for `react-router-dom`, the `<div>` wrapper is not necessary